_assets/js/click-tracking.js

```diff
@@ -0,0 +1,43 @@
+// Import our constants from constandts.js
+import { CLICK_TRACKING_ENDPOINT, ACCESS_KEY, AFFILIATE, MODULE_CODE } from './utils/constants';
+
+export default function clickTracking() {
+  // Get all the anchor tags that are children of b tags on the page:
+  const allLinks = document.querySelectorAll('.search-result');
+
+  // Loop through all the links and add a click event listener to each one:
+  allLinks.forEach((link, index) => {
+    const url = link.href;
+    const query = location.search.split('&').find((param) => param.startsWith('query='));
+    const position = location.search.split('&').includes('offset=') ?
+    Number(
+        location.search
+          .split('&')
+          .find((param) => param.startsWith('offset='))
+          .replace('offset=', '')
+      ) +
+      index +
+      1 : index + 1;
```

Comment:
Readability: This ternary statement is a bit hard to grok, wdyt about a `function getOffset()` that returns the offset or 0 if not found?

```suggestion
    const position = getOffset() + index + 1;
```